platform/darwin/src/MGLBackgroundStyleLayer.h

```diff
@@ -95,8 +95,8 @@ which it is added.
 /**
  The opacity at which the background will be drawn.
  
- The default value of this property is an expression that evaluates to the float
- 1. Set this property to `nil` to reset it to the default value.
+ The default value of this property is an expression that evaluates to the float 1.
+ Set this property to `nil` to reset it to the default value.
```

Comment:
This file is generated automatically by `make darwin-style-code`. The next time someone needs to update our runtime styling API, these changes will be overridden.

The script automatically wraps lines at a certain column. This function needs to be updated with some form of orphan control:

https://github.com/mapbox/mapbox-gl-native/blob/8c00a41708c6c77e0ef11fada874cdce1a985283/platform/darwin/scripts/generate-style-code.js#L56-L59

There may also be an NPM package we can depend on for this functionality.